packages/auth/docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
       - "5432:5432"
     environment:
       POSTGRES_PASSWORD: password
+      BYPASS_SIGNATURE_VALIDATION: false
```

Comment:
This could be worth an addition to the `docker-compose`s in the `infrastructure/local` folder as well, to make it obvious this can be flipped on and off in the local environment